### PR TITLE
Added 404 not found page.

### LINKED ISSUE #531: [UI] Custom 404 error page
**Description**
Currently, the below page appears on error 404 (page not found)
image
The above page should contain a message that the page could not be found.

**Expected Behavior**
A custom 404 error page similar to layer5
image

---
### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Zain-Muiz**
Signed-off-by: Zain-Muiz <zainu10863@gmail.com>

**Description**
A custom 404 error page similar to layer5 added.

This PR fixes #531 

**Notes for Reviewers**

Screenshot

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: _includes/pagenotfound.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      ><button class="btn">Return to Meshery</button></a
+    >
+  </div>
+</div>
+
+<script>
+  const messages = [
+    "Oh, no. Please pardon our meshy site.",
+    "Oops. Please excuse the mesh.",
+    "Looks like this page doesn't exists. What a mesh!",
```
[the remark this answers is not in the document]
Comment (Zain-Muiz) [reply _includes/pagenotfound.html:80]: Okay ! @warunicorn19

[the remark this answers is not in the document]
Comment (Zain-Muiz) [reply _includes/pagenotfound.html]: Have made the requested change @warunicorn19 .

**Review by Nikhil-Ladha — APPROVED** [review]
LGTM

**Review by warunicorn19 — COMMENTED** [review]
Nice👍

**Pull request merged**